addons/binding/org.openhab.binding.homematic/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Homematic Binding
+Bundle-SymbolicName: org.openhab.binding.homematic;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .
+Import-Package: com.google.common.collect,
+ org.apache.commons.httpclient,
```

Comment:
Do you really require Apache Commons Httpclient? Would it be a big change to move to the Jetty client? See this discussion: https://www.eclipse.org/forums/index.php/t/1069192/